packages/os/metricdog.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Send a Metricdog Ping
+
+[Service]
+Type=oneshot
+RemainAfterExit=false
+StandardError=journal+console
+ExecStart=/usr/bin/metricdog send-health-ping
```

Comment:
Can we set an overall timeout in the timer unit, or here (via `/usr/bin/timeout`)?

If the processes somehow get wedged in spite of our precautions, it would be good to prevent them from stacking up on the host.

Reply:
I think we want TimeoutStartSec https://unix.stackexchange.com/a/599394/296877

Reply:
[92230f5](https://github.com/bottlerocket-os/bottlerocket/compare/560f33678840b377569ea0dbb2d01633767172ae..92230f53e1f7aefca7c67130b785565a6d172824)

Added this here, will re-test just to make sure it still works.

Reply:
OK I've done the testing for the timeout. It works! When I add a 40 second sleep, I see that the service fails with a timeout message. Remove that sleep and it works.